### ISSUE TITLE
Empty <ul> in Social Media footer

### ISSUE THREAD

**Issue by robgasc** [ATLAS-Illinois/Illinois-Framework-Theme#1110]
**Describe the bug**
In the footer of some pages, there is an empty UL that is a placeholder for Social Media,. While it’s empty, it should either be commented out or removed.

**To Reproduce**
1. Go to ['Drupal Webtheme'](https://drupal.webtheme.illinois.edu/)
2. Open the inspector and inspect the footer just below the site title
3. See an empty social media ul present

**Expected behavior**
When no social media is added, there should be no social media elements in the footer.

**Additional context**
Original issue sourced by Campus Accessibility Checking. Pulled from LAS documentation of this issue.
